.github/workflows/cna.yml

```diff
@@ -0,0 +1,23 @@
+name: Create SumUp Next App
+
+on:
+  schedule:
+    - cron: '0 3 * * 1'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js v14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Bootstrap test app
+        run: yarn create next-app --example "https://github.com/sumup-oss/circuit-ui/tree/main/packages/create-sumup-next-app/template" test-app
+
+      - name: Build test app
```

Comment:
Is a build step enough, or should we start a dev server and assert for something, like the app name? Could there be errors that are not caught in build?

Reply:
A dev server would catch fewer errors because it compiles pages only when they’re loaded. A build could miss runtime errors, so we could start the server with `yarn start` and run assertions against the app 🤔 Not sure how to run two shell commands in parallel in GitHub Actions though.

Reply:
We can use [`start-server-and-test`](https://github.com/bahmutov/start-server-and-test), it's pretty common for running something like cypress in CI

Reply:
Oh, that's a cool tool! 🤩

Reply:
I've added a script to check the page title in 9d15b20261aaa09bade96ddced530623b77fb5e6, however, we won't be able to test if it actually works until we've merged all the recent changes, including this branch.